spark/benchmark.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+sudo apt-get update
+sudo apt-get -y install openjdk-8-jdk-headless
+
+# For Spark3.0.1 installation: 
+# wget --continue https://downloads.apache.org/spark/spark-3.0.1/spark-3.0.1-bin-hadoop2.7.tgz
```

Comment:
It does not reproduce:

```
ubuntu@ip-172-31-39-169:~$ wget --continue https://downloads.apache.org/spark/spark-3.0.1/spark-3.0.1-bin-hadoop2.7.tgz
--2024-02-01 22:37:09--  https://downloads.apache.org/spark/spark-3.0.1/spark-3.0.1-bin-hadoop2.7.tgz
Resolving downloads.apache.org (downloads.apache.org)... 135.181.214.104, 88.99.95.219, 2a01:4f8:10a:201a::2, ...
Connecting to downloads.apache.org (downloads.apache.org)|135.181.214.104|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2024-02-01 22:37:09 ERROR 404: Not Found.
```

Reply:
Will replace with `https://downloads.apache.org/spark/spark-3.5.0/spark-3.5.0-bin-hadoop3.tgz`